docs/package.json

```diff
@@ -11,72 +10,91 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
-	"type": "module",
 	"scripts": {
-		"build": "concurrently npm:build:api npm:build:md-magic && npm run hugo",
-		"build:api": "npm run download:api && npm run build:api-documentation",
-		"build:api-documentation": "node ./api-markdown-documenter/index.js",
-		"build:local": "concurrently npm:build:local:api npm:build:md-magic && npm run hugo",
-		"build:local:api": "npm run download:api && npm run build:local:rollup && npm run build:local:api-documentation",
-		"build:local:api-documentation": "node ./api-markdown-documenter/index.js true",
-		"build:local:rollup": "node ./local-api-rollup.js",
-		"build:md-magic": "markdown-magic",
-		"build:md-magic:code": "markdown-magic --files \"**/*.md\" !docs --workingDirectory ..",
-		"build:repo-docs": "npm run build:md-magic:code",
-		"ci:build": "npm run build",
-		"ci:linkcheck": "start-server-and-test ci:start 1313 linkcheck:full",
-		"ci:start": "http-server ./public --port 1313 --silent",
-		"clean": "rimraf --glob public content/docs/api _api-extractor-temp _doc-models",
-		"download:api": "node ./download-apis.js",
+		"build": "npm run build:generate-content && npm run build:site",
+		"build:api-documentation": "npm run download-doc-models && npm run generate-api-documentation",
+		"build:generate-content": "concurrently npm:build:markdown-magic npm:build:api-documentation",
+		"build:markdown-magic": "markdown-magic",
+		"prebuild:site": "npm run generate-versions",
+		"build:site": "docusaurus build",
```

Comment:
Somewhere here we need an invocation of `tsc` that uses `test/tsconfig.json`. Right now as far as I can tell nothing builds the tests, except maybe Playwright as it tries to run them?

Reply:
Yeah, Playwright handles transforming the TypeScript on our behalf. Probably a good idea to add an explicit build step for it, though. I'll do that.

Reply:
Added https://github.com/microsoft/FluidFramework/pull/22990/commits/997264b4c4d78a24b16f99c5c0975dd6d630aa8e